### PR TITLE
Do style

### PR DESCRIPTION
**Pull request by edahigure**
- Add any missing JSX elements to your project (without adding extra functionality)
- Your final design should match Zeplin's design (this includes all elements + the same styling)
- You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,23 +6,27 @@ import Categories from './pages/Categories';
...
+          <nav className="nav-bar">
+            <div className="Bookstore-CMS-2 ">
+              Bookstore CMS
+            </div>
 
+            <ul className="list-1">
+              <li><Link to="/books" className="li-item">BOOKS</Link></li>
+              <li><Link to="/categories" className="li-item li-item-2">CATEGORIES</Link></li>
+            </ul>
+          </nav>
```
Comment (muneebulrehman) [inline src/App.js]: - Optional
I recommend using `NavLink` instead of `Link` component.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @edahigure , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly remove the unrelevant sections from your README file to keep it clean and organized.
image

image



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Books.js [hunk 2]
```diff
@@ -36,48 +38,75 @@ export default function Books() {
...
+                title,
+                category,
+              }));
             }}
-          />
-        </label>
+          >
+            ADD BOOK
+          </button>
+        </div>
```
Comment (muneebulrehman) [inline src/pages/Books.js]: Kindly fix the styling for the `Add Book` button. 
Your design ⬇️
image

Provided design ⬇️
image

**Review by ghost — APPROVED** [review]
Hi @edahigure , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ App is designed as expected 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**